src/web/mjs/engine/Storage.mjs

```diff
@@ -802,6 +802,9 @@ export default class Storage {
             return path.replace( /\.+$/g, '' ).trim(); // remove trailing dots and whitespaces
         }
         if( this.platform.indexOf( 'linux' ) === 0 ) {
+            if(path.substring(path.length-1,path.length)==".") { //check if last character is a period
```

Comment:
This seems to be not robust enough, what if there are 2 or more dots at the end
Maybe use a regex instead, something like this:
`path.replace( /[\.\s]+$/, '' )`